src/style.css

```diff
@@ -0,0 +1,56 @@
+.search__bar {
+  position: relative;
+  display: flex;
+  align-items: center;
+}
+
+.search__bar--img {
+  width: 19px;
+  height: 19px;
+  position: absolute;
+  left: 26px;
+}
+
+.search__bar--field {
+  line-height: 70px;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 350;
```

Comment:
Font weight value `350` is not valid with modern browsers
See more https://www.w3schools.com/cssref/pr_font_weight.php

BTW - now you have good experience to see how browsers handle weird font-weights)